apps/backend/src/modules/tasks/task.controller.ts

```diff
@@ -240,6 +257,55 @@ class TaskController extends BaseController {
 			status: HTTPCode.OK,
 		};
 	}
+
+	/**
+	 * @swagger
+	 * /tasks/deadline/{id}:
```

Comment:
```suggestion
	 * /tasks/{id}/deadline:
```
This makes more sense to me